conf/modules/cv_qrcode.xml

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="qrcode" dir="computer_vision/qrcode">
+  <doc>
+    <description>
+      QR code reader
+    </description>
+  </doc>
+
+  <header>
+    <file name="qr_code.h"/>
+  </header>
+  <periodic fun="qrcode_periodic()" freq="2"/>
+  <makefile target="ap">
+    <file name="qr_code.c"/>
+
+    <!-- zbar -->
+    <configure name="EXTSW" value="$(PAPARAZZI_SRC)/sw/ext" />
+    <configure name="LIBZBAR" value="$(EXTSW)/libzbar/zbar-0.10" />
+
+    <file name="bch15_5.c" dir="$(LIBZBAR)/zbar/qrcode"/>
+    <file name="binarize.c" dir="$(LIBZBAR)/zbar/qrcode"/>
+    <file name="isaac.c" dir="$(LIBZBAR)/zbar/qrcode"/>
+    <file name="qrdec.c" dir="$(LIBZBAR)/zbar/qrcode"/>
+    <file name="qrdectxt.c" dir="$(LIBZBAR)/zbar/qrcode"/>
+    <file name="rs.c" dir="$(LIBZBAR)/zbar/qrcode"/>
+    <file name="util.c" dir="$(LIBZBAR)/zbar/qrcode"/>
+
+    <file name="qr_finder.c" dir="$(LIBZBAR)/zbar/decoder"/>
+    <file name="ean.c" dir="$(LIBZBAR)/zbar/decoder"/>
+
+
+    <file name="error.c" dir="$(LIBZBAR)/zbar"/>
+    <file name="image.c" dir="$(LIBZBAR)/zbar"/>
+    <file name="scanner.c" dir="$(LIBZBAR)/zbar"/>
+    <file name="symbol.c" dir="$(LIBZBAR)/zbar"/>
+    <file name="refcnt.c" dir="$(LIBZBAR)/zbar"/>
+    <file name="decoder.c" dir="$(LIBZBAR)/zbar"/>
+    <file name="img_scanner.c" dir="$(LIBZBAR)/zbar"/>
+
+    <!-- libexif flags -->
+    <define name="$(LIBZBAR)/zbar" type="include"/>
+    <define name="$(LIBZBAR)/include" type="include"/>
+
+    <!--define name="DEBUG_QR_FINDER" value="1"/-->
+
+    <flag name="CFLAGS" value="Wno-cast-qual"/>
```

Comment:
The only sensible solution would be to compile it as as lib separately with these warnings disabled and then link that lib...